Automattic/vip-go-mu-plugins
# Make sure --version arg gets passed when using network-wide

## Description
Recently we encountered a bug that when using `--network-wide` in combination with `--version`, the version would get ignored.



## Changelog Description

### Plugin Updated: Search

This change addresses an issue in `wp vip-search index` CLI command. In the following combination of arguments:
```
wp vip-search index --version=next --indexables=post --network-wide
```
`--version=next` would not get passed to individual runs per site. 


## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test

1. Adds a new version on multisite `wp vip-search index-versions add post --network-wide`
1. Run indexing `wp vip-search index --version=next --indexables=post --network-wide`
1. Validate counts to verify the ne version got correctly indexed for each site `wp vip-search health validate-counts --network-wide`
